src/BuildingRegistry.Projector/appsettings.json

```diff
@@ -14,8 +14,12 @@
     "Syndication": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.BuildingRegistry;Trusted_Connection=True;TrustServerCertificate=True;"
   },
 
+  "FeatureToggles": {
+    "UseProjectionsV2": false
+  },
+
   "DataDog": {
-    "Enabled": false,
+    "Enabled": true,
```

Comment:
why enable in the local appsettings.json?

Reply:
default should be `true` overwritten in our own to disable it.